ntbs-service/Models/Entities/Notification.cs

```diff
@@ -307,8 +287,8 @@ private string GetFormattedHealthcareSettingState()
         #endregion
         [AssertThat(@"ShouldValidateFull && HasDeathEventForPostMortemCase", ErrorMessage = ValidationMessages.DeathEventRequiredForPostMortemCase)]
         public bool HasDeathEventForPostMortemCase =>
-            ClinicalDetails.IsPostMortem != true || TreatmentEvents.Any(x =>
-                x.TreatmentEventTypeIsOutcome && x.TreatmentOutcome.TreatmentOutcomeType == TreatmentOutcomeType.Died);
+            ClinicalDetails.IsPostMortem != true || (TreatmentEvents != null && TreatmentEvents.Any(x =>
```

Comment:
The new job was complaining when trying to run .Any where TreatmentEvents weren't explicitly included as it has to be evaluated in memory, having this check doesn't seem like a bad thing but feel free to disagree!